### ISSUE TITLE
Max allowed header size too low in Broker Server

### ISSUE THREAD

**Issue by ankitsultana** [GoogleCloudPlatform/gcp-token-broker#21]
https://github.com/GoogleCloudPlatform/gcp-token-broker/blob/635aaf0d1357df409c289c536cfcf693a54eb385/apps/broker/src/main/java/com/google/cloud/broker/grpc/BrokerServer.java#L74

I suspect that the maximum header size is too low in the broker server. Currently, while creating the server we are not configuring `maxInboundMetadataSize`, so it defaults to 8 KB, which I think is too small, since the Kerberos token may exceed the 8 KB limit. (which is being added in the connector side as seen [here](https://github.com/GoogleCloudPlatform/gcp-token-broker/blob/master/connector/src/main/java/com/google/cloud/broker/hadoop/fs/BrokerGateway.java#L132))

I think it needs to be figured out what the appropriate size should be set to.

**Comment by jphalip** [comment]
Good catch. Do you recommend setting it to a specific value, or should we simply make it configurable?

**Comment by ankitsultana** [comment]
IIRC I doubled it for my use case, however the actual required size wasn't much more than 8 KB— but we can may be make it configurable.

I am not quite sure how long a Kerberos token may be in size. Do you know if there is a spec regarding it? If not, I'd say it's best to make it configurable.
